pkg/log/log.go

```diff
@@ -31,11 +31,11 @@ func (formatter *PiperLogFormatter) Format(entry *logrus.Entry) (bytes []byte, e
 
 	switch formatter.logFormat {
 	case logFormatDefault:
-		message = fmt.Sprintf("%-5s %-6s - %s\n", entry.Level, stepName, entry.Message)
+		message = fmt.Sprintf("%-5s %-6s - %s - %s\n", entry.Level, stepName, entry.Message, entry.Data[logrus.ErrorKey])
 	case logFormatWithTimestamp:
-		message = fmt.Sprintf("%s %-5s %-6s - %s\n", entry.Time.Format("15:04:05"), entry.Level, stepName, entry.Message)
+		message = fmt.Sprintf("%s %-5s %-6s - %s - %s\n", entry.Time.Format("15:04:05"), entry.Level, stepName, entry.Message, entry.Data[logrus.ErrorKey])
 	case logFormatPlain:
-		message = entry.Message + "\n"
+		message = fmt.Sprintf("%s - %s\n", entry.Message, entry.Data[logrus.ErrorKey])
```

Comment:
How does this look when there is no error? Always a trailing ´ - <nil>´ or similar?

Reply:
@stippi2: thanks, you are right --> changed that. Now we will get `(noErrorMessag)` similar to `stepName` where we get `(noStepName)` in case when no step name has been set.